./GenerateDatasetsXml.sh errors

When running the command `./GenerateDatasetsXml.sh` the following error is generated

```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at gov.noaa.pfel.erddap.GenerateDatasetsXml.<clinit>(GenerateDatasetsXml.java:45)
Caused by: java.lang.RuntimeException: ERROR during EDStatic Low Level Startup:
ERROR while reading /usr/local/tomcat/content/erddap/setup.xml: 
java.lang.RuntimeException: ERROR: You must change the <flagKeyKey> in setup.xml to a new, unique, non-default value. NOTE that this will cause the flagKeys used by your datasets to change. Any subscriptions using the old flagKeys will need to be redone.
```

This can be resolved by changing the value of **flagKeyKey** in  `erddap/content/setup.xml` and binding the path to /erddap/content when running the docker command

--- a/GenerateDatasetsXml.sh
+++ b/GenerateDatasetsXml.sh
@@ -2,5 +2,6 @@
 docker run --rm -it \
   -v "$(pwd)/datasets:/datasets" \
   -v "$(pwd)/logs:/erddapData/logs" \
+   -v $(pwd)/erddap/content:/usr/local/tomcat/content/erddap \
   axiom/docker-erddap:2.23-jdk17-openjdk \
   bash -c "cd webapps/erddap/WEB-INF/ && bash GenerateDatasetsXml.sh -verbose"

